src/main/java/com/unosquare/admin_core/back_end/repository/DashboardRepository.java

```diff
@@ -7,25 +7,28 @@
 import org.springframework.data.repository.query.Param;
 
 import java.time.LocalDate;
+import java.time.LocalDateTime;
 import java.util.List;
 
 
 public interface DashboardRepository extends JpaRepository<Event, Integer> {
     @Query(value = "SELECT e FROM Event e " +
-            "WHERE " +
+            "WHERE NOT( e.event_status_id = '4' AND e.last_modified < :cancelledFilteredTime) " +
+            "AND " +
             "((e.startDate BETWEEN :startDate AND :endDate) OR " +
             "(e.endDate BETWEEN :startDate AND :endDate) OR " +
             "(e.startDate > :startDate AND e.endDate < :endDate)) " +
             "AND " +
-            "e.employee.employeeId = :employeeId"
-    )
+            "e.employee.employeeId = :employeeId")
     List<Event> findCalendarMonthEventsForEmployee(@Param("employeeId") int employeeId,
                                                    @Param("startDate") LocalDate startDate,
-                                                   @Param("endDate") LocalDate endDate);
+                                                   @Param("endDate") LocalDate endDate,
+                                                   @Param("cancelledFilteredTime") LocalDateTime cancelledFilteredTime);
 
     @Query(value = "SELECT e FROM Event e " +
             "INNER JOIN Contract c on e.employee.employeeId = c.employee.employeeId " +
-            "WHERE " +
+            "WHERE NOT( e.event_status_id = '4' AND e.last_modified < :cancelledFilteredTime)" +
+            "AND " +
```

Comment:
Does this compile? It shouldn't... if it's not using native sql then the underscores won't work. It needs to follow the same pattern as the rest eventStatusId etc. not event_status_id.